python/graphstorm/config/argument.py

```diff
@@ -1450,11 +1450,11 @@ def _add_hyperparam_args(parser):
     # control evaluation
     group.add_argument("--eval-batch-size", type=int, default=argparse.SUPPRESS,
             help="Mini-batch size for computing GNN embeddings in evaluation.")
-    group.add_argument('--evaluation-frequency',
+    group.add_argument('--eval-frequency',
             type=int,
             default=argparse.SUPPRESS,
             help="How offen to run the evaluation. "
-                 "Every #evaluation_frequency iterations.")
+                 "Every #eval_frequency iterations.")
```

Comment:
Can we change the "eval_frequency" to "eval-frequency"?

Reply:
Fixed.